### PR TITLE
docs: add English version of cli

### LINKED ISSUE #1494: doc translation (Chinese to English): cli
- Translate the Chinese version to English: https://openmldb.ai/docs/zh/v0.4/reference/cli.html (you can download as a markdown file from the button at the top-right corner)
- Save the English version to docs/en/cli.md

### PR DESCRIPTION
**Pull request by cswjd**
docs: add English version of cli

### CODE REVIEW

**Comment by cswjd** [comment]
fixes #1494

File: docs/en/cli.md [hunk 1]
```diff
@@ -0,0 +1,475 @@
+# OpenMLDB operation CLI
```
Comment (lumianph) [inline docs/en/cli.md]: Operations in CLI
(sorry that this Chinese title is also not good)

File: docs/en/cli.md [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+
+* [ns client](#ns-client)
+* [tablet client](#tablet-client)
+
+### NS Client
```
Comment (lumianph) [inline docs/en/cli.md]: ## NS Client
(the Chinese doc also has this incorrect heading level issues, please correct this for the English doc)

File: docs/en/cli.md [hunk 3]
```diff
@@ -0,0 +1,475 @@
...
+
+Connecting to the ns client requires specifying zk\_cluster, zk\_root\_path and role. Where zk\_cluster is the zk address, zk\_root\_path is the root path of the cluster in zk, role is the role to be started and needs to be specified as ns_client
+
+```bash
+$ ./bin/openmldb --zk_cluster=172.27.2.52:12200 --zk_root_path=/onebox --role=ns_client
+```
+
+#### use
```
Comment (lumianph) [inline docs/en/cli.md]: ### use

File: docs/en/cli.md [hunk 4]
```diff
@@ -0,0 +1,475 @@
...
+  t1 21 0 172.24.4.55:9971 leader yes 1 1 301.000 9.353 K
+172.24.4.55:6531 demo_db> showtable auto_VCeOIIKA
+   name tid pid endpoint role is_alive offset record_cnt memused diskused
+-------------------------------------------------- -------------------------------------------------- --
+  auto_VCeOIIKA 25 0 172.24.4.55:9971 leader yes 8 4 498.000 9.128 K
+```
+
+#### showtablet
+
+View tablet information (if the serverName and automatic local ip function are used, the endpoint is serverName, and the real_endpoint is "-")
```
Comment (lumianph) [inline docs/en/cli.md]: (if the serverName and automatic local ip function are used, the endpoint is serverName, and the real_endpoint is "-")
=>
(if the serverName is specified and the local IP auto-configuration is enabled, then the endpoint is shown as the serverName and the real_endpoint is shown as "-")

File: docs/en/cli.md [hunk 5]
```diff
@@ -0,0 +1,475 @@
...
+  6534708420092481536 172.17.0.14:9533 kTabletHealthy 14h
+```
+
+#### addreplica
+
+Add replica
+
+Command format: `addreplica table_name pid_group endpoint`
+
+* table\_name table name
```
Comment (lumianph) [inline docs/en/cli.md]: table\_name: the table name

File: docs/en/cli.md [hunk 6]
```diff
@@ -0,0 +1,475 @@
...
+* pid\_group shard id collection. There can be the following situations
```
Comment (lumianph) [inline docs/en/cli.md]: pid\_group shard id collection.
=>
pid\_group: the collection of shard IDs

File: docs/en/cli.md [hunk 7]
```diff
@@ -0,0 +1,475 @@
...
+    * a single shard
+    * Multiple shards, separated by commas. Such as 1,3,5
```
Comment (lumianph) [inline docs/en/cli.md]: Multiple shard IDs, separated by commas, such as 1,3,5

File: docs/en/cli.md [hunk 8]
```diff
@@ -0,0 +1,475 @@
...
+    * Shard interval, the interval is a closed interval. For example, 1-5 means shard 1, 2, 3, 4, 5
```
Comment (lumianph) [inline docs/en/cli.md]: * A range of shard IDs with a closed interval; for example, 1-5 means shards 1, 2, 3, 4, 5

File: docs/en/cli.md [hunk 9]
```diff
@@ -0,0 +1,475 @@
...
+* endpoint The endpoint of the node to be added as a replica
```
Comment (lumianph) [inline docs/en/cli.md]: * endpoint: The endpoint of the node to be added as a replica

File: docs/en/cli.md [hunk 10]
```diff
@@ -0,0 +1,475 @@
...
+  endpoint real_endpoint state age
+-------------------------------------------------- ------------
+  6534708411798331392 172.17.0.12:9531 kTabletHealthy 1d
+  6534708415948603392 172.17.0.13:9532 kTabletHealthy 1d
+  6534708420092481536 172.17.0.14:9533 kTabletHealthy 14h
+```
+
+#### addreplica
+
+Add replica
```
Comment (lumianph) [inline docs/en/cli.md]: Add replicas

File: docs/en/cli.md [hunk 11]
```diff
@@ -0,0 +1,475 @@
...
+  test1 13 3 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+> addreplica test1 1,2,3 172.27.128.33:8541
+AddReplica ok
+> addreplica test1 1-3 172.27.128.33:8541
+AddReplica ok
+```
+
+#### delreplica
+
+Delete replica
```
Comment (lumianph) [inline docs/en/cli.md]: Delete replicas

File: docs/en/cli.md [hunk 12]
```diff
@@ -0,0 +1,475 @@
...
+
+Command format: `delreplica table_name pid_group endpoint`
+
+* table\_name table name
+* pid\_group shard id collection. There can be the following situations
+    * a single shard
+    * Multiple shards, separated by commas. Such as 1,3,5
+    * Shard interval, the interval is a closed interval. For example, 1-5 means shard 1, 2, 3, 4, 5
+* endpoint endpoint to delete replicas
```
Comment (lumianph) [inline docs/en/cli.md]: please replace those description as the same as the that of addreplica

File: docs/en/cli.md [hunk 13]
```diff
@@ -0,0 +1,475 @@
...
+  test1 13 3 172.27.128.33:8541 follower 0min yes kNoCompress 0 0 0.000
+> delreplica test1 1,2,3 172.27.128.33:8541
+DelReplica ok
+> delreplica test1 1-3 172.27.128.33:8541
+DelReplica ok
+```
+
+#### migrate
+
+copy migration
```
Comment (lumianph) [inline docs/en/cli.md]: Replicas migration

File: docs/en/cli.md [hunk 14]
```diff
@@ -0,0 +1,475 @@
...
+
+Command format: `migrate src_endpoint table_name pid_group des_endpoint`
+
+* src\_endpoint The node that needs to be checked out
+* table\_name table name
+* pid\_group shard id collection. There can be the following situations
+    * a single shard
+    * Multiple shards, separated by commas. Such as 1,3,5
+    * Shard interval, the interval is a closed interval. For example, 1-5 means shard 1, 2, 3, 4, 5
```
Comment (lumianph) [inline docs/en/cli.md]: please replace those description as the same as the that of addreplica

File: docs/en/cli.md [hunk 15]
```diff
@@ -0,0 +1,475 @@
...
+
+#### addreplica
+
+Add replica
+
+Command format: `addreplica table_name pid_group endpoint`
+
+* table\_name table name
+* pid\_group shard id collection. There can be the following situations
+    * a single shard
```
Comment (lumianph) [inline docs/en/cli.md]: A single shard

File: docs/en/cli.md [hunk 16]
```diff
@@ -0,0 +1,475 @@
...
+partition migrate ok
+```
+
+#### confget
+
+Get configuration information, currently only supports auto\_failover
+
+Command format: `confget [conf_name]`
+
+* conf\_name configuration item name, optional
```
Comment (lumianph) [inline docs/en/cli.md]: * conf\_name: 
(add a colon after all parameter names)

File: docs/en/cli.md [hunk 17]
```diff
@@ -0,0 +1,475 @@
...
+* value The value set by the configuration item
+
+```
+> confset auto_failover true
+set auto_failover ok
+```
+
+#### offlineendpoint
+
+Offline node. This command is asynchronous and after the successful return, you can view the running status through showopstatus
```
Comment (lumianph) [inline docs/en/cli.md]: Make a node offline. This command is asynchronous. After it succeeds, you can view the running status through `showopstatus`.

File: docs/en/cli.md [hunk 18]
```diff
@@ -0,0 +1,475 @@
...
+
+Command format: `offlineendpoint endpoint [concurrency]`
+
+* endpoint is the endpoint of the failed node. This command will perform the following operations on all shards under the node:
```
Comment (lumianph) [inline docs/en/cli.md]: endpoint: This is the given endpoint to go offline. This command will perform the following operations on all shards under the node:

File: docs/en/cli.md [hunk 19]
```diff
@@ -0,0 +1,475 @@
...
+  * If it is the master, execute the re-election of the master
```
Comment (lumianph) [inline docs/en/cli.md]: the master => a master

File: docs/en/cli.md [hunk 20]
```diff
@@ -0,0 +1,475 @@
...
+  * If it is a slave, find the master node and delete the current endpoint copy from the master node
+  * Modify is_alive status to no
+* concurrency controls the concurrent number of task execution. This configuration is optional, the default is 2 (name_server_task_concurrency configuration can be configured), and the maximum value is the value configured by name_server_task_max_concurrency
```
Comment (lumianph) [inline docs/en/cli.md]: concurrency => concurrency: (add a colon)

File: docs/en/cli.md [hunk 21]
```diff
@@ -0,0 +1,475 @@
...
+
+After the command is executed successfully, all shards will have leaders in the yes state
+
+#### recoverendpoint
+
+Restore node data. This command is asynchronous and after the successful return, you can view the running status through showopstatus
+
+Command format: `recoverendpoint endpoint [need_restore] [concurrency]`
+
+* endpoint is the endpoint of the node to restore
```
Comment (lumianph) [inline docs/en/cli.md]: * endpoint: the endpoint of the node to restore

File: docs/en/cli.md [hunk 22]
```diff
@@ -0,0 +1,475 @@
...
+* Whether the need_restore table topology is to be restored to the original state, this configuration is optional, the default is false. If set to true, a shard is the leader under this node, and it is still the leader after recoverendpoint is executed to restore data
```
Comment (lumianph) [inline docs/en/cli.md]: need_restore: Whether the table topology is to be restored ...

File: docs/en/cli.md [hunk 23]
```diff
@@ -0,0 +1,475 @@
...
+* concurrency controls the concurrent number of task execution. This configuration is optional, the default is 2 (name_server_task_concurrency configuration can be configured), and the maximum value is the value configured by name_server_task_max_concurrency
```
Comment (lumianph) [inline docs/en/cli.md]: add a colon after "concurrency"

**Review by lumianph — CHANGES_REQUESTED** [review]

File: docs/en/cli.md (hunk 10 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 11 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 12 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 23 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 22 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 21 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 20 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 19 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 18 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 17 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 16 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 15 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 14 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 9 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 8 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 7 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 6 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 5 above)
Comment (L-Y-L) [reply docs/en/cli.md]: change to:
 table_name: The table name
ok?

File: docs/en/cli.md (hunk 4 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 3 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 2 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 1 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md (hunk 5 above)
Comment (cswjd) [reply docs/en/cli.md]: Ok, thank you.

File: docs/en/cli.md [hunk 24]
```diff
@@ -51,18 +51,18 @@ View tablet information (if the serverName and automatic local ip function are u
...
 ```
 
-#### addreplica
+### addreplica
 
-Add replica
+Add replicas
 
-Command format: `addreplica table_name pid_group endpoint`
+Command format: `addreplica the table name pid_group endpoint`
```
Comment (lumianph) [inline docs/en/cli.md]: this might be my mistake, your original word is correct. sorry. pls help change it back
the table name => table_name

**Review by lumianph — CHANGES_REQUESTED** [review]
thanks, pls help address one more comment

File: docs/en/cli.md (hunk 24 above)
Comment (cswjd) [reply docs/en/cli.md]: I'm careless, thank you.

**Review by lumianph — APPROVED** [review]

**Pull request merged**